CHANGELOG.md

```diff
@@ -5,8 +5,11 @@
         - Plugins supported: `table`
     - API:
       - Support for exits numbers and names. New member `exits` in `RouteStep`, based on `junction:ref` on ways
+      - `RouteStep` now has new parameter `classes` that can be set in the profile on each way.
```

Comment:
This is not about road classes? This flag is user-defined, it doesn't expose our road-class hierarchy. The classes are mostly for two things:

1/ Announce to the user when we enter a new class (e.g. private road or toll road)
2/ Begin able to mark a step as using toll roads/private roads whatever in the UI

Main reason I don't want is as annotation is that at the moment we just use data that we already need for the avoid flags anyway (edge-based-node level), for annotations we need segment-level which means a lot more data.